crds/enterprise.gloo.solo.io_v1_crds.yaml

```diff
@@ -1,8 +1,10 @@
 # Code generated by skv2. DO NOT EDIT.
 
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    crd.solo.io/specHash: 2ffe0f9b23ee9b70
```

Comment:
why does this specHash get added to everything?

Reply:
This is done as part of skv2 codegen. There is a `skipSpecHash` field (https://github.com/solo-io/skv2/blob/10d7a529e381d354047e6f91286890a82b342caf/codegen/model/resource.go#L66) which I didn't opt out of. Since these CRDs aren't even used, I figured it was ok, but more than happy to skip them

Reply:
56a9bb5c08e125373227e643be735bac2e809ee7